apluslms_roman/builder.py

```diff
@@ -18,7 +19,6 @@ def __init__(self, engine, config, observer=None):
         self._engine = engine
         self._observer = observer or StreamObserver()
 
-
```

Comment:
Drop this change.

Reply:
In PEP: 
Method definitions inside a class are surrounded by a single blank line.
https://www.python.org/dev/peps/pep-0008/#blank-lines
Shouldn't we make it only one blank line?

Reply:
My bad. I think I looked this was in root. Yeah, it's a correct change and I didn't notice this in earlier PR (when it was introduced).